### ISSUE TITLE
[Bug] Cannot rewind subgraph that has failed

### ISSUE THREAD

**Issue by paymog** [graphprotocol/graph-node#4861]
### Bug report

Almost a year ago I deployed a subgraph. Now I see that it's errored out and I'm trying to rewind it but I get the following error:

```
 ❯❯❯ graphman rewind --block-hash 0xd4fbd0df522604031fead6b69ce8127949ee6b06cf0e58c6e4408bb59916ec42 --block-number 108380746 QmXv1itQZThauznuJk2tWUzvK8uCHAeYbZ5vokL89LvPXy --force
Pausing deployments

Rewinding deployments
Error: store error: Deployment details not found for QmXv1itQZThauznuJk2tWUzvK8uCHAeYbZ5vokL89LvPXy: internal constraint violated: the hash and number of a block pointer must either both be null or both have a value, but for `QmXv1itQZThauznuJk2tWUzvK8uCHAeYbZ5vokL89LvPXy` the hash of start_block is `None` and the number is `Some(BigDecimal(0))`
command terminated with exit code 1
```

Is this a new invariant that's been added to graph node in the past few versions? What's the proper way to remediate this considering the subgraph has been running for months without issue?

### IPFS hash

QmXv1itQZThauznuJk2tWUzvK8uCHAeYbZ5vokL89LvPXy

### Some information to help us out

- [ ] Tick this box if this bug is caused by a regression found in the latest release.
- [ ] Tick this box if this bug is specific to the hosted service.
- [X] I have searched the issue tracker to make sure this issue is not a duplicate.

### OS information

None

**Comment by azf20** [comment]
👀 can you confirm the version of Graph Node you are running? @incrypto32 @lutter any ideas if things have changed here in graphman or elsewhere?

**Comment by incrypto32** [comment]
I cant recall any changes for rewind other than the way subgraphs were paused for rewind, but that shouldn't cause this.

@paymog so what are the steps to reproduce this?

Just trying to rewind `QmXv1itQZThauznuJk2tWUzvK8uCHAeYbZ5vokL89LvPXy` after its synced upto failure ?

**Comment by paymog** [comment]
Whoops, missed your comments. It seems like this particular subgraph had some invalid value for the `subgraphs.subgraph_manifest` table with a null value for the `start_block_hash` column. I have no idea how it got into this state and don't know how to reproduce.

I manually fixed this by looking up the hash for the start block and updating the row for this subgraph in the `subgraph_manifest` table. Once I did that, I could rewind the subgraph.
